### PR TITLE
Add the missing documentation for the Rest API

### PR DESCRIPTION
**Pull request by MartinSchoeler**
Will add the following missing API methods to the documentation:
### Channel
  - [x] addOwner
  - [x] addModerator
  - [x] removeOwner
  - [x] removeModerator

### Group
  - [x] addOwner
  - [x] addModerator
  - [x] removeOwner
  - [x] removeModerator

### Im
  - [x] setTopic
  - [x] history
  - [x] list
  - [x] list.everyone
  - [x] close
  - [x] open

@graywolf336 if you see something wrong or if someone else already done any of the items above, please let me know

### CODE REVIEW

**Review by graywolf336 — DISMISSED** [review]
Make sure you add them to the list of method calls on each `README.md` that is inside the folder of each type.

**Comment by MartinSchoeler** [comment]
@graywolf336 did i miss anything or there is something wrong?

File: 6. Developer Guides/REST API/Im/messages.others.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+# Im Messages.others
+Retrieves the messages from any direct message in the server, this method supports the [Offset and Count Query Parameters](../Offset%20and%20Count%20Info.md).
+
+For this method work the `Enable Direct Message History Endpoint` setting must be true, and the user calling this method must have the `view-room-administration` permission.
+
+| URL | Requires Auth | HTTP Method |
+| :--- | :--- | :--- |
+| `/api/v1/im.messages.others` | `yes` | `GET` |
+
+## Payload
```
Comment (graywolf336) [inline 6. Developer Guides/REST API/Im/messages.others.md]: When the http method is `GET` change this header to be `Query Parameters`.

**Review by graywolf336 — CHANGES_REQUESTED** [review]

File: 6. Developer Guides/REST API/Im/messages.others.md (hunk 1 above)
Comment (MartinSchoeler) [reply 6. Developer Guides/REST API/Im/messages.others.md]: @graywolf336  Fixed! Also fixed the other errors like this one in some other files

File: 6. Developer Guides/REST API/Im/README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+order: 40
+---
+
+# Im Methods
```
Comment (graywolf336) [inline 6. Developer Guides/REST API/Im/README.md]: Capitalize the `IM` for all of the documentation for it.

File: 6. Developer Guides/REST API/Im/README.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+| :--- | :--- | :--- |
+| `/api/v1/im.close` | Removes a direct message from the list of direct messages. | [Link](close.md) |
+| `/api/v1/im.history` | Retrieves the messages from a direct message. | [Link](history.md) |
+| `/api/v1/im.messages.others` | Retrieves the messages from any direct message in the server. | [Link](messages.others.md) |
+| `/api/v1/im.list` | List the direct messages the caller is part of. | [Link](list.md) |
+| `/api/v1/im.list.everyone` | List all direct message the caller in the server. | [Link](list.everyone.md) |
+| `/api/v1/im.open` | Adds the direct message back to the list of direct messages. | [Link](open.md) |
+| `/api/v1/im.setTopic` | Sets a direct message topic. | [Link](setTopic.md) |
+
+From version 0.50.0 and on you can call the methods using `dm` instead of `im`.
```
Comment (graywolf336) [inline 6. Developer Guides/REST API/Im/README.md]: Add a heading for this, maybe `## Notes`

File: 6. Developer Guides/REST API/Im/list.everyone.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
+---
+order: 20
+---
+
+# Im List.Eeveryone
```
Comment (graywolf336) [inline 6. Developer Guides/REST API/Im/list.everyone.md]: `IM List Everyone`

File: 6. Developer Guides/REST API/Im/list.everyone.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+Lists all of the direct messages in the server, this method supports the [Offset and Count Query Parameters](../Offset%20and%20Count%20Info.md).
```
Comment (graywolf336) [inline 6. Developer Guides/REST API/Im/list.everyone.md]: Add a note about the permission this method requires.

File: 6. Developer Guides/REST API/Im/messages.others.md [hunk 6]
```diff
@@ -0,0 +1,88 @@
+---
+order: 25
+---
+
+# Im Messages.others
```
Comment (graywolf336) [inline 6. Developer Guides/REST API/Im/messages.others.md]: `IM Messages Others`

File: 6. Developer Guides/REST API/Im/messages.others.md [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+Retrieves the messages from any direct message in the server, this method supports the [Offset and Count Query Parameters](../Offset%20and%20Count%20Info.md).
+
+For this method work the `Enable Direct Message History Endpoint` setting must be true, and the user calling this method must have the `view-room-administration` permission.
```
Comment (graywolf336) [inline 6. Developer Guides/REST API/Im/messages.others.md]: Missing a **to** before `work`.

File: 6. Developer Guides/REST API/Im/open.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
+---
+order: 30
+---
+
+# Channel Open
```
Comment (graywolf336) [inline 6. Developer Guides/REST API/Im/open.md]: This is direct messages and not channels.

**Review by graywolf336 — CHANGES_REQUESTED** [review]

**Review by graywolf336 — APPROVED** [review]

**Pull request merged**